SeventhHeavenUI/Classes/GameLauncher.cs

```diff
@@ -1682,9 +1683,13 @@ private void CollectCrashReport()
                 // === 7th files ===
                 archive.AddEntry("7thWrapperLoader.log", Path.Combine(Sys.InstallPath, "7thWrapperLoader.log"));
                 archive.AddEntry("applog.txt", File.Open(Sys.PathToApplog, FileMode.Open, FileAccess.Read, FileShare.ReadWrite));
-                archive.AddEntry("profile.xml", Sys.PathToCurrentProfileFile);
                 archive.AddEntry("settings.xml", Sys.PathToSettings);
                 archive.AddEntry("registry_transaction.bat", Path.Combine(Sys.PathToTempFolder, "registry_transaction.bat"));
+                // Convert profile.xml to profile.txt
+                Profile currentProfile = Util.Deserialize<Profile>(Sys.PathToCurrentProfileFile);
+                IEnumerable<string> profileDetails = currentProfile.GetDetails();
+                File.WriteAllLines(Path.Combine(Sys.PathToTempFolder, "profile.txt"), profileDetails);
+                archive.AddEntry("profile.txt", Path.Combine(Sys.InstallPath, "profile.txt"));

```

Comment:
Here should be also `Sys.PathToTempFolder`

Reply:
🤦‍♂️ whoops